### ISSUE TITLE
Component is broken due to change by Panasonic

### ISSUE THREAD

**Issue by pensionado** [djbulsink/panasonic_ac#21]
Since a few days setup fails with these errors:
`Sep 18 08:21:58 dirksehome2 docker/hass[884]: 2020-09-18 08:21:58 ERROR (MainThread) [homeassistant.components.climate] Error while setting up panasonic_ac platform for climate
Sep 18 08:21:58 dirksehome2 docker/hass[884]: Traceback (most recent call last):
Sep 18 08:21:58 dirksehome2 docker/hass[884]:   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 179, in _async_setup_platform
Sep 18 08:21:58 dirksehome2 docker/hass[884]:     await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
Sep 18 08:21:58 dirksehome2 docker/hass[884]:   File "/usr/local/lib/python3.8/asyncio/tasks.py", line 483, in wait_for
Sep 18 08:21:58 dirksehome2 docker/hass[884]:     return fut.result()
Sep 18 08:21:58 dirksehome2 docker/hass[884]:   File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
Sep 18 08:21:58 dirksehome2 docker/hass[884]:     result = self.fn(*self.args, **self.kwargs)
Sep 18 08:21:58 dirksehome2 docker/hass[884]:   File "/config/custom_components/panasonic_ac/climate.py", line 76, in setup_platform
Sep 18 08:21:58 dirksehome2 docker/hass[884]:     for device in api.get_devices():
Sep 18 08:21:58 dirksehome2 docker/hass[884]:   File "/usr/local/lib/python3.8/site-packages/pcomfortcloud/session.py", line 174, in get_devices
Sep 18 08:21:58 dirksehome2 docker/hass[884]:     for device in group['deviceIdList']:
Sep 18 08:21:58 dirksehome2 docker/hass[884]: KeyError: 'deviceIdList'
`
Apparently this is caused by a change in JDSON from panasonic, pcomfortcloud 0.16 will fix this(issue #30 deviceIdList error in session.py). Can you let me know how to install this.

**Comment by SeraphimSerapis** [comment]
Providing another log

```
Logger: homeassistant.components.climate
Source: custom_components/panasonic_ac/climate.py:76
Integration: Climate (documentation, issues)
First occurred: September 17, 2020, 8:42:45 PM (1 occurrences)
Last logged: September 17, 2020, 8:42:45 PM

Error while setting up panasonic_ac platform for climate
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 193, in _async_setup_platform
    await asyncio.shield(task)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/panasonic_ac/climate.py", line 76, in setup_platform
    for device in api.get_devices():
  File "/usr/local/lib/python3.8/site-packages/pcomfortcloud/session.py", line 174, in get_devices
    for device in group['deviceIdList']:
KeyError: 'deviceIdList'
```

I'll look into fixing this over the weekend.

Tim

**Comment by shyne99** [comment]
Fixed by pull request #20 already merged. Issue can be closed
